### PR TITLE
NMS-10393: There is no documentation on the instrumentation log reader

### PR DESCRIPTION
**Pull request by Bonrob2**
NMS-10393: There is no documentation on the instrumentation log reader - added documentation in the admin UI section to describe this feature.

### All Contributors

* [ ] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [ ] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://issues.opennms.org) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the JIRA issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here,
or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-10393

### CODE REVIEW

File: docs/modules/operation/pages/admin/webui/log-reader.adoc [hunk 1]
```diff
@@ -0,0 +1,17 @@
+[[log-statistics]]
+= Instrumentation Log Statistics
+
+The instrumentation log reader determines how long data collection takes for each node in the system, and which nodes are taking the most time.
+You can view this information in the Instrumentation Log Statistics UI (menu:Gears icon[Additional Tools>Instrumentation Log Reader].
+
+The instrumentation log reader requires instrumentation logs produced with a DEBUG level.
```
Comment (mmahacek) [inline docs/modules/operation/pages/admin/webui/log-reader.adoc:7]: The instrumentation log is defaulted to INFO, which is enough to provide data.

Comment (Bonrob2) [reply docs/modules/operation/pages/admin/webui/log-reader.adoc:7]: Thanks. The wiki content was more of a spec. Must have changed in the actual development.

File: docs/modules/operation/pages/admin/webui/log-reader.adoc [hunk 2]
```diff
@@ -0,0 +1,17 @@
+[[log-statistics]]
+= Instrumentation Log Reader
+
+The instrumentation log reader provides information on how long data collection takes for each node in the system, and which nodes are taking the most time.
+You can view this information in the Instrumentation Log Statistics UI (menu:Gears icon[Additional Tools>Instrumentation Log Reader].
+
+By default, the instrumentation log level is INFO.
+The instrument log reader reads each line of the file to determine the time at which events took place during data collection, and uses that information to compute the following information:
```
Comment (mmahacek) [inline docs/modules/operation/pages/admin/webui/log-reader.adoc:8]:
```suggestion
The reader parses through the `instrumentation.log` file generated by Collectd.
Each line of the file is analyzed to determine the time at which events took place during data collection, and uses that information to compute the following information:
```

File: docs/modules/operation/pages/admin/webui/log-reader.adoc [hunk 3]
```diff
@@ -0,0 +1,17 @@
+[[log-statistics]]
+= Instrumentation Log Reader
+
+The instrumentation log reader provides information on how long data collection takes for each node in the system, and which nodes are taking the most time.
+You can view this information in the Instrumentation Log Statistics UI (menu:Gears icon[Additional Tools>Instrumentation Log Reader].
```
Comment (mmahacek) [inline docs/modules/operation/pages/admin/webui/log-reader.adoc:5]: missing closing parenthesis
```suggestion
You can view this information in the Instrumentation Log Statistics UI (menu:Gears icon[Additional Tools>Instrumentation Log Reader]).
```

Comment (Bonrob2) [reply docs/modules/operation/pages/admin/webui/log-reader.adoc:5]: Argh! Good catch!

**Review by mmahacek — APPROVED** [review]
ship it!

**Pull request merged**